packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -234,13 +213,36 @@ function selectorFor_useRoomInfo(
   };
 }
 
-export function selectInboxNotifications<M extends BaseMetadata>(
-  state: UmbrellaStoreState<M>
-): InboxNotificationData[] {
-  const result = applyOptimisticUpdates(state);
-  return Object.values(result.inboxNotifications).sort(
-    // Sort so that the most recent notifications are first
-    (a, b) => b.notifiedAt.getTime() - a.notifiedAt.getTime()
+/**
+ * @private Do not rely on this internal API.
+ */
+// TODO This helper should ideally not have to be exposed at the package level!
+// TODO It's currently used by react-lexical though.
+export function selectThreads<M extends BaseMetadata>(
+  state: UmbrellaStoreState<M>,
```

Comment:
We can look into removing this dependency from `react-lexical` if it helps with the refactor. We used `selectThreads` in `react-lexical` because there wasn't a better alternative then, but with the refactor, maybe we could expose an API (that also aligns with the current refactor) on the store that allows us to remove the `selectThreads` import.

Reply:
I think we're getting closer to removing this, yes. Ideally we wouldn't need a selector and we could just read the stable result from the store, but pagination will soon require changes here, so for now we're good.